src/obb_rules/serializer/reader.cljc

```diff
@@ -0,0 +1,161 @@
+(ns ^{:added "2.1.0"
+      :author "Pedro Santos"}
+  obb-rules.serializer.reader
+  "Loads and processes a game from a serialized text format"
+  (:require [clojure.string :as string]
+            [obb-rules.serializer.common :as common]
+            [obb-rules.game :as game]
+            [obb-rules.result :as result]
+            [obb-rules.board :as board]
+            [obb-rules.turn :as turn]
+            [obb-rules.host-dependent :as host]
+            [obb-rules.game-mode :as game-mode]))
+
+(defmulti str->action
+  "Converts a action in string format to a raw action"
+  (fn [s]
+    (first s)))
+
+(defn get-int
+  "Given a string, gets the char on the specific position, and returns it
+  as an integer"
+  [s pos]
+  (host/parse-int (str (nth s pos))))
+
+(defmethod str->action \a
+  [s]
+  [:attack [(get-int s 1) (get-int s 2)]
+           [(get-int s 3) (get-int s 4)]])
```

Comment:
Couldn't 1, 2, 3 and 4 be made constants? I think this would make this functions more readable, since I would be able to identify immediately what is being obtained from the string representation. Another option would be to wrap each call to get-int (`(get-int s 1)`) in a function with the exact name of the thing being obtained.